File: benchmarks/py/array_fill.py
import sys


def array_fill(n: int) -> int:
  arr = [0] * n
  for i in range(n):
    arr[i] = i
  s = 0
  for i in range(n):
    arr[i] += 1
    s += arr[i]
  return s


if __name__ == "__main__":
  n = int(sys.argv[1]) if len(sys.argv) > 1 else 0
  print(array_fill(n))

File: benchmarks/js/array_fill.js
function arrayFill(n) {
  const arr = new Array(n);
  for (let i = 0; i < n; i++) {
    arr[i] = i;
  }
  let sum = 0;
  for (let i = 0; i < n; i++) {
    arr[i] = arr[i] + 1;
    sum += arr[i];
  }
  return sum;
}

const n = process.argv[2] ? parseInt(process.argv[2]) : 0;
console.log(arrayFill(n));
